commands/plugin/install.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const spawn = require('child-process-ext/spawn');
+const fsp = require('fs').promises;
+const fse = require('fs-extra');
+const path = require('path');
+const _ = require('lodash');
+const isPlainObject = require('type/plain-object/is');
+const yaml = require('js-yaml');
+const cloudformationSchema = require('@serverless/utils/cloudformation-schema');
+const log = require('@serverless/utils/log');
+const ServerlessError = require('../../lib/serverless-error');
+const yamlAstParser = require('../../lib/utils/yamlAstParser');
+const fileExists = require('../../lib/utils/fs/fileExists');
+const npmCommandDeferred = require('../../lib/utils/npm-command-deferred');
+const CLI = require('../../lib/classes/CLI');
+const {
+  getPlugins,
+  getPluginInfo,
+  getServerlessFilePath,
+  validate,
+} = require('../../lib/commands/plugin-management');
+
+const cli = new CLI(undefined);
+
+module.exports = async ({ configuration, serviceDir, configurationFilename, options }) => {
+  const pluginInfo = getPluginInfo(options.name);
+  options.pluginName = pluginInfo[0];
+  options.pluginVersion = pluginInfo[1] || 'latest';
+
+  validate({ serviceDir });
+  const plugins = await getPlugins();
+  const plugin = plugins.find((item) => item.name === options.pluginName);
+  if (!plugin) {
+    cli.log('Plugin not found in serverless registry, continuing to install');
+  }
+
+  await pluginInstall({ configuration, serviceDir, options });
```

Comment:
Let's confirm whether plugin is not already installed (it needs to be in both `configuration` and in `package.json`)